libs/application/templates/rental-agreement/src/fields/PropertySearch/index.tsx

```
@@ -0,0 +1,447 @@
+    Record<string, number>
+  >(storedValue?.numOfRooms || {})
+
+  const [selectedAddress, setSelectedAddress] = useState<
+    AddressProps | undefined
+  >(storedValue)
+  const [propertiesByAddressCode, setPropertiesByAddressCode] = useState<
+    HmsPropertyInfo[] | undefined
+  >(storedValue?.propertiesByAddressCode || [])
+
+  useEffect(() => {
+    if (selectedAddress && selectedAddress.addressCode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)